### PR TITLE
some changes in menu

### PR DESCRIPTION
**Pull request by sylma2**

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -1,30 +1,40 @@
+nav {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 80%;
+  margin: auto;
+}
 p {
```
Comment (Macio1992) [inline style.css]: Please be more specific with this selector: `nav p`

File: style.css [hunk 2]
```diff
@@ -1,30 +1,40 @@
...
-  display: inline;
+  margin: 0;
   font-size: 50px;
   font-family: "Lucida Sans", "Lucida Sans Regular", "Lucida Grande",
```
Comment (Macio1992) [inline style.css]: This should be in general selector at the top of the file -> `* { font-family ... }`

File: style.css [hunk 3]
```diff
@@ -1,30 +1,40 @@
...
     "Lucida Sans Unicode", Geneva, Verdana, sans-serif;
-  text-align: left;
-  margin-left: 50px;
   font-weight: bolder;
```
Comment (Macio1992) [inline style.css]: I think `bold` is enough

File: style.css [hunk 4]
```diff
@@ -1,30 +1,40 @@
...
 }
-nav ul {
-  display: inline;
-  text-align: justify;
-  justify-content: right;
+
+nav ul li {
+  padding: 30px;
 }
 nav span {
```
Comment (Macio1992) [inline style.css]: Please be more specific with this selector. `nav p span`

File: style.css [hunk 5]
```diff
@@ -1,30 +1,40 @@
...
   color: blue;
 }
-li.button {
+button {
```
Comment (Macio1992) [inline style.css]: More specific `nav ul li button`

File: style.css [hunk 6]
```diff
@@ -1,30 +1,45 @@
...
   color: blue;
 }
-li.button {
+
+nav ul li button {
   background-color: rgb(43, 164, 204);
-  padding: 2%;
-  border-radius: 10px;
+  border-radius: 5px;
+  text-decoration: none;
```
Comment (Macio1992) [inline style.css:40]: you can remove it

File: style.css [hunk 7]
```diff
@@ -1,30 +1,45 @@
...
+  padding: 20px;
+  border: rgb(43, 164, 204);
```
Comment (Macio1992) [inline style.css]: no border needed :) `border:none`

File: style.css [hunk 8]
```diff
@@ -1,30 +1,45 @@
...
+  color: #232340;
+  font-size: 18px;
```
Comment (Macio1992) [inline style.css:44]: please add `cursor: pointer` to this button

File: index.html [hunk 9]
```diff
@@ -10,11 +10,11 @@
...
-        <li><a href=""></a> Product</li>
-        <li><a href=""></a> Blog</li>
-        <li><a href=""></a> Support</li>
-        <li><a href=""></a> Log In</li>
-        <li class="button"><a href=""></a> Get Access</li>
+        <li><a href="">Product</a> </li>
+        <li><a href="">Blog</a> </li>
+        <li><a href=""> Support</a> </li>
+        <li><a href="">Log In</a> </li>
+        <li> <button>Get Access</button></li>
```
Comment (Macio1992) [inline index.html]: Please format both files :)

**Review by Macio1992 — APPROVED** [review]

File: style.css (hunk 6 above)
Comment (sylma2) [reply style.css:40]: czegoś takiego u mnie nie ma

**Pull request merged**